src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs

```diff
@@ -167,51 +170,66 @@ public static void SetCursorColor(EditText editText, Color color)
 						PrepareFields(editText.Context);
 					}
 
-					if (_cursorDrawableField == null || _cursorDrawableResField == null || _editorField == null || PorterDuff.Mode.SrcIn == null)
+					if (PorterDuff.Mode.SrcIn == null)
 					{
-						// PrepareFields() failed, give up now
 						editText.Log().WarnIfEnabled(() => "Failed to change the cursor color. Some devices don't support this.");
 						return;
 					}
 
-					var mCursorDrawableRes = _cursorDrawableResField.GetInt(editText);
-					var editor = _editorField.Get(editText);
-
-#if __ANDROID_28__
+					if ((int)Build.VERSION.SdkInt >= 29)
+					{
 #pragma warning disable 618 // SetColorFilter is deprecated
```

Comment:
Have you found out what the replacement for SetColorFilter is from the android documentation ?

Reply:
We'll  use the proposed BlendModeColorFilter which is the one proposed by Android documentation instead.
https://developer.android.com/reference/android/graphics/BlendModeColorFilter